Documentation/devicetree/bindings/iio/dac/adi,max22007.yaml

```
@@ -0,0 +1,107 @@
+# SPDX-License-Identifier: (GPL-2.0 OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/iio/dac/adi,max22007.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Analog Devices MAX22007 Quad Channel Industrial Analog Output
+
+maintainers:
+  - Janani Sunil <janani.sunil@analog.com>
+
+description: |
+  The MAX22007 is a quad-channel, 12-bit digital-to-analog converter (DAC)
+  with integrated precision output amplifiers and current output capability.
+  Each channel can be independently configured for voltage or current output.
+
+  Datasheet available at:
+    https://www.analog.com/en/products/max22007.html
+
+properties:
+  compatible:
+    const: adi,max22007
+
+  reg:
+    maxItems: 1
+
+  '#address-cells':
+    const: 1
+
+  '#size-cells':
+    const: 0
+
```

How do you power the device?
```suggestion

  vdd-supply:
    description:
      Vdd analog supply voltage from +2.7V to +5.5V.

  hvdd-supply:
    description:
      HVdd positive high-voltage power supply from +8V to (HVSS +24V) for the
      output channels.
      
  #... others
```
datasheet p16
for reference
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/iio/dac/adi%2Cad5770r.yaml